geode-protobuf/src/test/java/org/apache/geode/protocol/protobuf/operations/PutAllRequestOperationHandlerJUnitTest.java

```diff
@@ -14,6 +14,14 @@
  */
 package org.apache.geode.protocol.protobuf.operations;
 
+import static org.mockito.Mockito.any;
```

Comment:
Have you updated your IntelliJ style guide file with the June 13 version?  They've modified the order of imports somewhat.